iothub/device/src/Transport/Amqp/ClientWebSocketTransport.cs

```diff
@@ -8,10 +8,10 @@
 using System.Net.WebSockets;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Azure.Amqp;
+using Microsoft.Azure.Amqp.Transport;
 
-using Microsoft.Azure.Devices.Client;
-
-namespace Microsoft.Azure.Amqp.Transport
+namespace Microsoft.Azure.Devices.Client
 {
     internal sealed class ClientWebSocketTransport : TransportBase, IDisposable
```

Comment:
Will the cases of `TaskContinuationOptions.ExecuteSynchronously` be changed to `TaskContinuationOptions.RunContinuouslyAsynchronously` in this or a follow-up PR?

Also, the TCS ctors need to be changed to take the same parameter.

See https://github.com/davidfowl/AspNetCoreDiagnosticScenarios/blob/master/AsyncGuidance.md#always-create-taskcompletionsourcet-with-taskcreationoptionsruncontinuationsasynchronously